aiohttp_devtools/runserver/watch.py

```diff
@@ -108,7 +109,13 @@ def _start_process(self):
         act = 'Start' if self._change_count == 0 else 'Restart'
         logger.info('%sing dev server at http://%s:%s ●', act, self._config.host, self._config.main_port)
 
-        self._process = Process(target=serve_main_app, args=(self._config,))
+        try:
+            tty_path = os.ttyname(sys.stdin.fileno())
```

Comment:
I have no idea if this is a reliable way of getting the stdin tty path. It works for me on ubuntu.

Reply:
`os.ttyname` is not available on Windows but works on any Unix system IMHO.